src/main/java/com/tcs/java8/utility/FizzBuzzUtils.java

```diff
@@ -0,0 +1,18 @@
+package com.tcs.java8.utility;
+
+public class FizzBuzzUtils {
+
+    public static String getFizzBuzzResponse(int inputNumber) throws RuntimeException
+    {
+      if(inputNumber<0)
+          throw new RuntimeException("NEGETIVE NUMBER IS NOT ALLOWED");
```

Comment:
Custom Exception should be made rather or IllegalArgumentException should be thrown rather then passing message in RuntimeException